pom.xml

```diff
@@ -206,7 +206,7 @@
                 </executions>
                 <configuration>
 <!--                     Uncomment to see detailed container logs -->
-<!--                     <showLogs>true</showLogs> -->
+                    <showLogs>true</showLogs>
```

Comment:
The showLogs is for debugging. It is an exception (in my opinion) to the 'never check in commented out code' rule. Normally it should be left commented out - in some projects the Docker logs can exceed the log length for Travis CI.